lib/srv/app/session.go

```diff
@@ -162,6 +163,10 @@ func (s *Server) newStreamWriter(identity *tlsca.Identity) (events.StreamWriter,
 			User:         identity.Username,
 			Impersonator: identity.Impersonator,
 		},
+		AppMetadata: apievents.AppMetadata{
+			AppPublicAddr: app.GetPublicAddr(),
+			AppName:       app.GetName(),
+		},
```

Comment:
Let's include `AppURI` here and everywhere else too. This way, if the app is deleted and added with the same name but pointing to a different web application, you'll still be able tell them apart in the events.